types/node/module.d.ts

```diff
@@ -283,7 +283,7 @@ declare module "module" {
         }
     }
     interface RegisterOptions<Data> {
-        parentURL: string | URL;
+        parentURL?: string | URL;
```

Comment:
According to he [NodeJS document](https://nodejs.org/docs/latest-v18.x/api/module.html#moduleregisterspecifier-parenturl-options), parentURL has a default value of `data:`, so it's always optional.